app/views/admin/users/_reactions.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="row my-3">
+  <div class="col-12">
+    <h2 class="d-inline">Recent Reactions</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="col-12 collapse" id="reactions-row">
+    <div class="list-group-flush">
+    <% unless @related_vomit_reactions.empty? %>
+      <% @related_vomit_reactions.each do |reaction| %>
+        <a href="<%= reaction.reactable.path %>" class="list-group-item list-group-item-action">
```

Comment:
This layout is simple and it works, but I wonder if it wouldn't be more helpful to Forem admins to have this information be more **visual**, kinda like the `Vomit Reactions` table under `/admin/reports`?

So if it is a vomit record, it displays a 🤢 emoji. I'm not sure if it's only vomits we are tracking?

Also since the record is a link, have it appear in a link color (blue).

And I think we should include the author name for the article (or whatever was vommitted on)

<img width="1178" alt="Screen Shot 2020-08-28 at 2 04 31 PM" src="https://user-images.githubusercontent.com/32520970/91601763-354e8480-e938-11ea-95c9-aecb7a436bb3.png">

Reply:
These are nice suggestions @msarit! I actually styled everything this way per @Zhao-Andy's request on the issue itself, so maybe Andy would have some more insight on whether or not these stylistic changes are necessary. As for adding the author name, I'm not sure if that would add unnecessary data to the view -- we want to be able to see what was vomited on and who the vomit was for, but I don't know if we need to include the author name, since theoretically, the Admin should be able to click through to the content that was vomited on and gather extra data there!

Reply:
I think @msarit's style suggestions are nice! I think it would be worth updating in this PR.

I agree we don't need the author name since these reactions are for the author already.

Reply:
Agreed and making these now, as well as taking into account the logic for `reactable_type: "User"`!